azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunner.java

```diff
@@ -278,7 +278,9 @@ private void setupFlowExecution() {
       this.watcher.setLogger(logger);
     }
 
-    logger.info("Assigned executor : " + AzkabanExecutorServer.getApp().getExecutorHostPort());
+    if (AzkabanExecutorServer.getApp() != null) {
+      logger.info("Assigned executor : " + AzkabanExecutorServer.getApp().getExecutorHostPort());
+    }
```

Comment:
This is enough to make the test work – otherwise there's no need to have that initialized instance of `AzkabanExecutorServer` available via the static getter.